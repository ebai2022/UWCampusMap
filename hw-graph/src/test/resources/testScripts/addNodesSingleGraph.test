# Test adding nodes to a singular graph

# Create a graph
CreateGraph A

# Add nodes
AddNode A e1
AddNode A e2

# Check if the nodes have been added
ListNodes A