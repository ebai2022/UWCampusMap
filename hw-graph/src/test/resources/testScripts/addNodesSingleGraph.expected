# Test adding nodes to a singular graph

# Create a graph
created graph A

# Add nodes
added node e1 to A
added node e2 to A

# Check if the nodes have been added
A contains: e1 e2